WordPress/WordPress.xcodeproj/xcshareddata/xcschemes/WordPress Internal.xcscheme

```diff
@@ -81,11 +86,17 @@
             ReferencedContainer = "container:WordPress.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-debugPinghub"
+            isEnabled = "NO">
+         </CommandLineArgument>
+      </CommandLineArguments>
       <EnvironmentVariables>
          <EnvironmentVariable
-            key = "NSZombieEnabled"
-            value = "YES"
-            isEnabled = "YES">
+            key = "OS_ACTIVITY_MODE"
+            value = "disable"
+            isEnabled = "NO">
```

Comment:
Oh actually I left the option there but disabled it so if we need to turn it on it's there for us to turn on/off.